### PR TITLE
docs(Form): add required and width examples

### LINKED ISSUE #1444: Add `required` Form.Field example
Our `Form Variations` section is missing the `required` form field example:
https://semantic-ui.com/collections/form#required

We should add this example in the same fashion as the current inline variation example.

### PR DESCRIPTION
**Pull request by layershifter**
Fixes #1444. Also adds example with withds.

### CODE REVIEW

File: docs/app/Examples/collections/Form/FieldVariations/FormExampleRequiredField.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <Form>
+    <Form.Field required>
+      <label>Last name</label>
+      <Input placeholder='Full name' />
+    </Form.Field>
+    <Form.Checkbox
+      inline
+      label='I agree to the terms and conditions'
+      required
+    />
```
Comment (levithomason) [inline docs/app/Examples/collections/Form/FieldVariations/FormExampleRequiredField.js]: Could we split these into two examples with separate descriptions?
***
A Form.Field can be required.
```jsx
<Form.Field required>
```

Form.Field shorthand can also be required.
```jsx
<Form.Checkbox ... required />
```

**Comment by layershifter** [comment]
@levithomason I've splitted examples as you asked 👍

**Comment by levithomason** [comment]
Beautiful!  Will merge on pass 👍

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.68.0`.

**Pull request merged**